schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(50)
```

Comment:
- [ Required ] I think it would be nice if you add the `species` column to the `animals` table through the `ALTER TABLE` command. In a production environment, it might not be possible to edit the `SCHEMA` of a table directly like you've done here. Please see the example below for a suggestion on how to achieve this.

## Example code
```SQL
ALTER TABLE animals ADD species varchar(255);
```